EIPS/eip-5625.md

```diff
@@ -0,0 +1,204 @@
+---
+eip: 5625
+title: NFT Metadata JSON Schema dStorage Extension
+description: Add a dStorage property to non-fungible tokens (NFTs) metadata JSON schema to provide decentralized storage information of NFT assets
+author: Gavin Fu (@gavfu)
+discussions-to: https://ethereum-magicians.org/t/eip-5625-nft-metadata-json-schema-dstorage-extension/10754
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-08
+requires: 721, 1155
+---
+
+## Abstract
+
+This standard is an extension to NFT metadata JSON schema defined by [EIP-721](./eip-721.md) and [EIP-1155](./eip-1155.md). As a complement, it provides additional information about how the NFT asset is stored in a decentralized storage system, if applicable.
```

Comment:
```suggestion
This EIP extends the NFT metadata JSON schema defined in [EIP-721](./eip-721.md) and [EIP-1155](./eip-1155.md), adding a `dStorage` key that provides information about how the NFT data is stored. 
```
I recommend you use this rephrasing of your current abstract. As written, the second sentence is grammatically incorrect.